test/unittest/dds/subscriber/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ if(NOT ((MSVC OR MSVC_IDE) AND EPROSIMA_INSTALLER))
 
         add_executable(DataReaderTests ${DATAREADERTESTS_SOURCE})
         target_compile_definitions(DataReaderTests PRIVATE FASTRTPS_NO_LIB
+            BOOST_ASIO_STANDALONE
+            ASIO_STANDALONE
```

Comment:
No need of these, as we are linking fastrtps (see my next comment)

Reply:
Without that compiler is trying to find a boost header file.

Reply:
Done